dev-docs/modules/userId.md

```diff
@@ -21,6 +21,7 @@ The User ID module supports multiple ways of establishing pseudonymous IDs for u
 
 * **BritePool ID** - Britepool Identity Resolution userId submodule. Universal Identity resolution which does not depend on 3rd party cookies.
 * **Criteo ID for Exchanges** –  specific id for Criteo and its partners that enables optimal take rate on all web browsers.
+* **ID+** - a universal identity solution that aims to empower the marketing ecosystem with a privacy-compliant mechanism to overcome identity resolution challenges arising from ID fragmentation, cookie restrictions and increasingly stringent regulation across the world. It is powered by Zeotap and built on the back of the world’s largest marketing identity graph.
```

Comment:
Is Zeotap also the owner of the marketing graph? If so, can reword this to: 

Zeotap and utilizes its extensive marketing identity graph. 

We try to stay neutral on products and making any kind of claims such as largest, biggest, fastest, etc.

Reply:
Thanks for the suggestion here @MartianTribe , have updated the documentation as per the expectations.